Makefile

```diff
@@ -1,3 +1,5 @@
+PREFIX := $(DESTDIR)/usr
```

Comment:
DESTDIR isn't part of the prefix, it's part of the install target. ie if we use $(PREFIX)/share/clear-containers as a string in the binary, we don't want DESTDIR in there.

```
PREFIX=/usr
...
install:
        install -D $(TARGET) $(DESTDIR)/$(BINDIR)/$(TARGET)
```